crates/typst/src/visualize/color.rs

```diff
@@ -105,7 +105,12 @@ const ANGLE_EPSILON: f32 = 1e-5;
 ///
 /// # Predefined color maps
 /// Typst also includes a number of preset color maps that can be used for
-/// gradients. Most of these color maps are chosen to be color blind friendly.
```

Comment:
Why remove the color blindness mention?

Reply:
Just because it's already explained under the table. Broke the flow to have it twice.

Reply:
Got it, thanks for the explanation :slightly_smiling_face: